runtime/driver/src/main/java/org/apache/nemo/driver/NemoDriver.java

```diff
@@ -149,7 +153,18 @@ private void shutdown() {
     @Override
     public void onNext(final StartTime startTime) {
       setUpLogger();
-      runtimeMaster.requestContainer(resourceSpecificationString);
+      if (executorType.equals("lambda")) {
+        LOG.info("##### lambda executor #####");
+        runtimeMaster.requestLambdaExecutor();
```

Comment:
this code just invokes one lambda executor. We need to create multiple lambda executors in the future. Please add todo

Reply:
Added.